packages/blocks/src/api/test/validation.js

```diff
@@ -727,8 +727,6 @@ describe( 'validation', () => {
 				'Apples'
 			);
 
-			expect( console ).toHaveWarned();
```

Comment:
Do you know why there are no longer validation warnings and errors reported in unit tests here?

Reply:
Yes, I removed the warnings from the `isBlockContentValid` function because it's a function returning a boolean, it's not like we triggered something and we want to see logs (like parsing)

Reply:
Cool, it makes sense now that you explained it. Test descriptions fit the new behavior so 💯